src/transformers/tokenization_auto.py

```diff
@@ -174,9 +174,12 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
         if "bert-base-japanese" in pretrained_model_name_or_path:
             return BertJapaneseTokenizer.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
 
-        for config_class, tokenizer_class in TOKENIZER_MAPPING.items():
+        for config_class, (tokenizer_class_py, tokenizer_class_ru) in TOKENIZER_MAPPING.items():
```

Comment:
maybe `tokenizer_class_rust` or `tokenizer_class_fast`? (when i see `_ru` I think it's a russian tokenizer :)